dvc/command/check_ignore.py

```diff
@@ -61,9 +106,24 @@ def add_parser(subparsers, parent_parser):
         help="Show the target paths which don’t match any pattern. "
         "Only usable when `--details` is also employed",
     )
+    parser.add_argument(
+        "--stdin",
+        action="store_true",
+        default=False,
+        help="Read pathnames from the standard input, one per line, "
+        "instead of from the command-line.",
+    )
+    parser.add_argument(
+        "-a",
+        "--all",
+        action="store_true",
+        default=False,
+        help="Show all of the patterns match the target paths. "
+        "Only usable when `--details` is also employed",
```

Comment:
"Show all target paths and whether they match a pattern or not."

Reply:
I'd also move this option before -n, since it seems more useful IMO.

Reply:
> Show all target paths

Hmmm. I see that's not exactly what this option does. It shows all the patterns that match each target indeed so maybe just:

"Show all the patterns that match each target path."

And is there any way to have a --details list with both matched and not matched targets?

Reply:
> And is there any way to have a --details list with both matched and not matched targets?

I see -n already does this 🙂 — will probably update in docs.